src/VSCodeExtension/src/generator-qsharp/app/index.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+const Generator = require('yeoman-generator');
+const chalk = require('chalk');
+const yosay = require('yosay');
```

Comment:
I'd recommend replacing calls to require with the use of TypeScript's import statements, as t hat will be more portable to mixing different module systems.

Reply:
Sounds good. As I'm adding the next round of changes in index.ts (to replace the scaffolding by the real thing) I'll fix the imports.